.env.base

```diff
@@ -72,3 +72,7 @@ REACT_APP_MTPELERIN_BUY_URL=https://buy.mtpelerin.com/
 REACT_APP_MTPELERIN_SELL_URL=https://sell.mtpelerin.com/
 # TODO(0xdef1cafe) - update this when we have a real referral code
 REACT_APP_MTPELERIN_REFERRAL_CODE=shapeshift
+
```

Comment:
can we please add a feature flag to this file too? :)

Reply:
I've replaced the existing MtPelerin FF since I've seen it's been released - unless there's a reason to keep it?

Reply:
Would prefer for this to be its own line diff and not remove the existing flag @lmyslinski - for blame purposes

Reply:
@lmyslinski plz add this as a separate flag - we'll remove all the old flags in separate PRs